package.json

```diff
@@ -56,7 +56,7 @@
   "binary": {
     "module_name": "bcrypt_lib",
     "module_path": "./lib/binding/",
-    "host": "https://github.com",
-    "remote_path": "/kelektiv/node.bcrypt.js/releases/download/v{version}/"
+    "host": "https://github.com/kelektiv/node.bcrypt.js/releases/download/",
+    "remote_path": "v{version}"
```

Comment:
Does Travis allows us to retrieve artefacts from a build job? 
There are some compiled binaries for Linux and OSX at my fork generated by travis. 
And for windows, I could not understand how things work on AppVeyor